src/app/comments/newCommentForm/NewCommentForm.jsx

```diff
@@ -1,8 +1,10 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import { postComment, getLocation } from '../../../actions';
-import { selectLocationData, selectIsPostingComment } from '../../../reducers';
+import { selectIsPostingComment } from '../../../reducers';
+import { getLocationError, getLocation as selectLocationData } from '../../../selectors/location';
```

Comment:
There was already a `getLocation` selector in `selectors/location`. I aliased it to `selectLocationData` in some modules, where it was necessary to do so because of a naming conflict with `actions/getLocation`, but it might be better to change it at the source, inside of `selectors/location`.

Reply:
Ah, I see.
Then yeah, it would definitely be good to rename getLocation at the source. But either way, if we want the convention to be "select" going forward, we shouldn't be introducing new selectors with the "get" convention. (I.e. getLocationError.)

Reply:
This is fixed in 3c253d8599bfbf6c2e803ce05fe0a7ec1b1c05d8